### PR TITLE
Persist grass to neo4j sync

### PR DESCRIPTION
**Pull request by pe4cey**
Fixes a known regression where users' grass files are no longer syncs with the neo4j browser sync service.

### CODE REVIEW

File: src/browser/modules/Sync/BrowserSync.jsx [hunk 1]
```diff
@@ -264,6 +264,12 @@ export class BrowserSync extends Component {
 
     return (
       <Drawer id='sync-drawer'>
+        <button
+          onClick={() =>
+            this.props.sendActionToDispatch({ type: 'grass/SYNC_GRASS' })}
+        >
+          Sync grass
+        </button>
```
Comment (oskarhane) [inline src/browser/modules/Sync/BrowserSync.jsx]: Why do we have this button here? Sync happens automatically, right?

Comment (pe4cey) [reply src/browser/modules/Sync/BrowserSync.jsx]: 👍

**Comment by pe4cey** [comment]
PR updated

**Comment by oskarhane** [comment]
There seems to be an issue where style changes are spammed to the sync service, hogging down the Browser completely.
Also, sync changes doesn't seem to be picked up after 20 revisions are reached.

**Pull request merged**